packages/components/src/Password/Reset.purs

```diff
@@ -0,0 +1,68 @@
+module KSF.Password.Reset where
+
+import Prelude
+
+import Data.Either (Either(..))
+import Data.Maybe (Maybe(..), fromMaybe)
+import Effect (Effect)
+import Effect.Aff (launchAff_)
+import Effect.Class (liftEffect)
+import KSF.Password.SendLink (requestResetLink)
+import KSF.Password.Change (updatePasswordForm)
+import KSF.Tracking as Tracking
+import KSF.User (User)
+import KSF.User as User
+import React.Basic.Hooks (Component, component, useEffectOnce, useState', (/\))
+import React.Basic.Hooks as React
+import React.Basic.DOM as DOM
+
+foreign import getCode :: Effect String
```

Comment:
Could we use https://pursuit.purescript.org/packages/purescript-web-url/1.0.2/docs/Web.URL.URLSearchParams#v:get ?